### ISSUE TITLE
Mistral-related warning messages in nightly container

### ISSUE THREAD

**Issue by torsli** [NVIDIA-NeMo/Automodel#947]
**Describe the bug**

When running benchmark scripts in nemo-automodel:nightly, I get warning messages regarding mistral:

`🚨 Something went wrong trying to find the model name in the path: /opt/Automodel/nemo_automodel/components/models/mistral3/model.py`
`🚨 Config not found for model. You can manually add it to HARDCODED_CONFIG_FOR_MODELS in utils/auto_docstring.py`
`🚨 `loss` is part of CausalLMOutputWithPast.__init__'s signature, but not documented. Make sure to add it to the docstring of the function in <string>.`
`🚨 `logits` is part of CausalLMOutputWithPast.__init__'s signature, but not documented. Make sure to add it to the docstring of the function in <string>.`

**Steps/Code to reproduce bug**

These occur when I import the arg parser method:

`from nemo_automodel.components.config._arg_parser import parse_cli_argv`

**Expected behavior**

No warning messages.

**Additional context**

This did not occur in the benchmarking scripts run on 12/6 so I figure it's related to the latest PRs.

**Comment by qiaochuz-nv** [comment]
## QA RCCA Analysis

**Fix PR:** #946
**Root Cause:** Spurious warning about model name path parsing for Mistral3 model in nightly container.

**Coverage:** COVERED — `automodel_mistral_7b_chunked_ce_ddp_test.sh` exercises Mistral model. Cosmetic warning fix with minimal regression risk.

**Conclusion:** No new test needed.
